templates/shared/_blender-contact-us-form.html

```diff
@@ -0,0 +1,77 @@
+<section class="p-strip is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1>{{h1}}</h1>
+      <p>{{intro_text}}</p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <form action="https://pages.ubuntu.com/index.php/leadCapture/save" method="post" id="mktoForm_{{formid}}">
+
+        <fieldset class="u-no-margin--bottom">
+          <label for="CommentsAboutProject" class="p-heading--three">Tell us about your project and what problem(s) you’re looking to solve.</label>
+          <input required id="CommentsAboutProject" name="CommentsAboutProject" maxlength="255" type="textarea" class="mktoField mktoRequired" />
```

Comment:
- The input doesn't render correctly - textarea is an element, not an input type
- I _think_ `Comments_from_lead__c` is a field name set in Marketo, or it's at least used in a lot of places throughout u.c
```suggestion
          <label for="Comments_from_lead__c" class="mktoLabel p-heading--three">Tell us about your project and what problem(s) you’re looking to solve:</label>
          <textarea required id="Comments_from_lead__c" name="Comments_from_lead__c" rows="4" class="mktoField mktoRequired" maxlength="2000"></textarea>
```